DataFormats/SoATemplate/README.md

```diff
@@ -129,6 +133,39 @@ using SoA1Layout = SoA1LayoutTemplate<>;
 using SoA1LayoutAligned = SoA1LayoutTemplate<cms::soa::CacheLineSize::defaultSize, cms::soa::AlignmentEnforcement::enforced>;
 ```
 
+It is possible to declare methods that operate on the SoA elements:
+
+```C++
+#include "DataFormats/SoALayout.h"
+
+GENERATE_SOA_LAYOUT(SoATemplate,
+  SOA_COLUMN(double, x),
+  SOA_COLUMN(double, y),
+  SOA_COLUMN(double, z),
+  
+  // methods operating on const_element
+  SOA_CONST_METHODS(
+    auto norm() {
```

Comment:
How about recommending the `const_element` member functions to be `const` along
```suggestion
    auto norm() const {
```
? (also in the tests)

I guess the non-mutability is mostly implied, but to make sure that these functions cane be called from `const const_element`.